akka-docs/src/main/paradox/typed/coexisting.md

```diff
@@ -87,10 +87,10 @@ Java
 
 
 @scala[That adds some implicit extension methods that are added to classic and typed `ActorSystem`, `ActorContext` and `ActorRef` in both directions.]
-@java[To convert between typed and classic `ActorSystem`, `ActorContext` and `ActorRef` in both directions there are adapter methods in `akka.actor.typed.javadsl.Adapter`.]
+@java[To convert between typed and classic `ActorSystem`, `ActorContext` and `ActorRef` in both directions there are adapter methods in @javadoc[akka.actor.typed.javadsl.Adapter](akka.actor.typed.javadsl.Adapter).]
 Note the inline comments in the example above. 
 
-This method of using a top level classic actor is the suggested path for this type of co-existence. However, if you prefer to start with a typed top level actor then you can use the @scala[implicit `spawn` -method]@java[`Adapter.spawn`] directly from the typed system:
+This method of using a top level classic actor is the suggested path for this type of co-existence. However, if you prefer to start with a typed top level actor then you can use the @scala[implicit `spawn` -method]@java[@javadoc[Adapter.spawn](akka.actor.typed.javadsl.Adapter#spawn(akka.actor.ActorSystem,akka.actor.typed.Behavior,java.lang.String,akka.actor.typed.Props))] directly from the typed system:
```

Comment:
I'm not sure about `implicit spawn`. Which link should be?

Reply:
I think that is referring to https://github.com/akka/akka/blob/main/akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/adapter/package.scala#L61